prac_05/emails.py

```diff
@@ -0,0 +1,53 @@
+"""Extract name from email, confirm user name, print name and email."""
+
+# Estimated time: 1.5 hours
+# Actual time: 4 hours
+
+SPECIAL_CHARACTERS = '@'
+
+
+def main():
+    """Extract name from email, confirm user name, print name and email."""
+    emails_and_names = {}
+
+    email = input("Email: ").strip()
+    email = get_valid_email(email)
+    while email != '':
+        name = get_name_from_email(email)
+        is_name_correct = input(f"Is this your name {name}? (Y/n): ").strip().lower()
+        if is_name_correct == 'n' or is_name_correct == 'no':
+            name = input("Enter your name: ").strip().title()
+        elif is_name_correct == '' or is_name_correct == 'Y':
```

Comment:
The input has a .lower() meaning the is_name_correct == 'Y' will never be True.
From naming conventions i might assume is_name_correct would be a boolean value not a string.